tests/frontendtest.cpp

```diff
@@ -28,3 +28,18 @@ TEST_F(FrontendTest, PrintEvaluatedCSV) {
 	std::string result = frontend.csvStream->str();
 	EXPECT_EQ(expected, result);
 }
+
+TEST_F(FrontendTest, SpecificPlotting) {
```

Comment:
Kunne du ikke også lave en unit test der giver fejlen in case at man giver et kemikale der ikke er i systemet? Det er base `ASSERT_THROW(statement, std::runtime_error)`. Ellers sidder den lige i skabet.

Reply:
jeg har legit lige skrevet en test til not equal mens du skrev den her kommentar

Reply:
oh, sådan en test. Sure